lib/jnpr/junos/facts/get_software_information.py

```diff
@@ -17,6 +17,12 @@ def _get_software_information(device):
             except Exception:
                 pass
         try:
+            # JDM for Junos Node Slicing
+            return device.rpc.cli("show version all-servers", format='xml',
```

Comment:
by this time we dont if any data in facts to do "if" check in place of try/catch.

Also we can just use below rpc in place of cli
```
<rpc>
        <get-software-information>
        <all-servers/>
        </get-software-information>
</rpc>